# Error Message
user.existed.exception = {0}\ub294 \uc874\uc7ac\ud558\ub294 \uc544\uc774\ub514\uc785\ub2c8\ub2e4.
user.notfound.exception = {0}\ub294 \uc874\uc7ac\ud558\uc9c0 \uc54a\ub294 \uc544\uc774\ub514\uc785\ub2c8\ub2e4.
password.mismatch.exception = {0} \uc0ac\uc6a9\uc790\uc758 \ube44\ubc00\ubc88\ud638\uac00 \uc77c\uce58\ud558\uc9c0 \uc54a\uc2b5\ub2c8\ub2e4.

instance.not.init={0} \uc778\uc2a4\ud134\uc2a4\uac00 \ucd08\uae30\ud654\ub418\uc9c0 \uc54a\uc558\uc2b5\ub2c8\ub2e4.
email.not.send=Exception\uc774 \ubc1c\uc0dd\ud558\uc5ec \uba54\uc77c\uc744 \uc804\uc1a1\ud558\ub294 \uc911 \uc5d0\ub7ec\uac00 \ubc1c\uc0dd\ud588\uc2b5\ub2c8\ub2e4.
original.error.message=Original \uc5d0\ub7ec\uba54\uc138\uc9c0\ub294 \ub2e4\uc74c\uacfc \uac19\uc2b5\ub2c8\ub2e4.

404.error={0}\ub97c \ucc3e\uc744 \uc218 \uc5c6\uc2b5\ub2c8\ub2e4.!!
505.error=\uc2dc\uc2a4\ud15c \uc5d0\ub7ec\uac00 \ubc1c\uc0dd\ud588\uc2b5\ub2c8\ub2e4.!!